pkg/handler/aws/uploadProfilePicture/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/CollActionteam/collaction_backend/auth"
+	"github.com/CollActionteam/collaction_backend/internal/uploads"
+	awsRepository "github.com/CollActionteam/collaction_backend/pkg/repository/aws"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+func handler(ctx context.Context, req events.APIGatewayV2HTTPRequest) (res events.APIGatewayProxyResponse, err error) {
```

Comment:
use `events.APIGatewayV2HTTPResponse`

Reply:
Can you kindly explain why?

Reply:
we are using `events.APIGatewayV2HTTPRequest` for req and the equivalent of this in response type is `events.APIGatewayV2HTTPResponse`. 
no critical requirement to change it.

Reply:
@Sobowale-Olayiwola This still needs to be adressed